api/src/services/import-export.ts

```diff
@@ -132,29 +130,20 @@ export class ImportService {
 				return await service.upsertOne(value, { bypassEmitAction: (action) => nestedActionEvents.push(action) });
 			});
 
+			const transform = (value: any, _field: string) => {
+				if (value === 'NULL') return null;
```

Comment:
Adding the following to the transformer solves the type error:

```suggestion
				if (value === 'NULL') return null;
        if (value === '') return null;
```